src/components/Table/components/Body/Body.module.scss

```diff
@@ -0,0 +1,54 @@
+.tr {
+  border-bottom: 1px solid $color-7;
+  position: relative;
+  transform: scale(1, 1);
```

Comment:
dlaczego to jest potrzene? O ile dobrze myślę, to nic to nie skaluje

Reply:
pozycjonowanie absolutne w tabeli, taki hack